services/tasks/src/main/java/org/hyperledger/besu/services/tasks/InMemoryTaskQueue.java

```diff
@@ -32,6 +32,9 @@ public class InMemoryTaskQueue<T> implements TaskCollection<T> {
   private final Set<InMemoryTask<T>> unfinishedOutstandingTasks = new HashSet<>();
   private final AtomicBoolean closed = new AtomicBoolean(false);
 
+  /** Default constructor. */
+  public InMemoryTaskQueue() {}
```

Comment:
Why is there a need for a default constructor here ? it should be implicit or there're changes related to generics in Java 21 that makes them mandatory ?

Reply:
@ahamlat Hi Ameziane, We have javadoc lint enabled in our gradle build and fail our build on lint warnings. One of the "lint" warning that has been enabled since Java 19 is to provide javadoc on default constructors.

https://bugs.openjdk.org/browse/JDK-8249634

To avoid adding such empty constructors, we have to disable gradle build failure on javadoc lint warnings.

Reply:
My opinion would to disable gradle build failure on javadoc lint warnings, and not add "dead code", but this is not blocking.

Reply:
I had a look, looks like we have `-Werror` in java compile task, this seems to apply to javadoc task as well.